hack/tools/go.mod

```
@@ -1,6 +1,6 @@
 module github.com/operator-framework/rukpak/hack/tools
 
-go 1.17
+go 1.18
 
 require (
 	github.com/golangci/golangci-lint v1.45.1
```

Bump this version to match the version we're using in github actions:
```suggestion
	github.com/golangci/golangci-lint v1.45.2
```

---

@joelanford Seems I missed out on that. I've pushed the update